node/cn/gasprice/gasprice.go

```diff
@@ -342,12 +339,16 @@ func (oracle *Oracle) getBlockValues(ctx context.Context, blockNum uint64, limit
 // paying any tip is unnecessary. It returns true when the head block is after Magma fork
 // and the next base fee is at the lower bound.
 func (oracle *Oracle) isRelaxedNetwork(header *types.Header) bool {
-	pset, err := oracle.gov.EffectiveParams(header.Number.Uint64() + 1)
-	if pset != nil && err == nil {
-		nextBaseFee := misc.NextMagmaBlockBaseFee(header, pset.ToKIP71Config())
-		return nextBaseFee.Cmp(big.NewInt(int64(pset.LowerBoundBaseFee()))) <= 0
+	pset := oracle.govModule.EffectiveParamSet(header.Number.Uint64() + 1)
```

Comment:
It's repeating quite a lot. Do you have either plan:
- Change `NextMagmaBlockBaseFee(header, pset)`
- Add `pset.ToKip71Config()`

Reply:
The former involves lots of changes, so fixed with the latter. aa01753